requirements.txt

```diff
@@ -0,0 +1,77 @@
+argparse==1.2.1
+backports.ssl-match-hostname==3.4.0.2
+Beaker==1.3.1
+cactusboards-amc13-python==1.1.11
+cactuscore-tsxdaqclient==2.6.3
+cactuscore-uhal-gui==2.3.0
+cactuscore-uhal-pycohal==2.4.0
+cas==0.15
+chardet==2.2.1
+cups==1.0
+cupshelpers==1.0
+decorator==3.0.1
+ethtool==0.6
+firstboot==1.110
+freeipa==2.0.0a0
+gmpy==1.15
+gyp==0.1
+iniparse==0.3.1
+iotop==0.3.2
+ipapython==3.0.0
+ipython==0.13.2
+iwlib==1.0
+javapackages==1.0.0
+kerberos==1.1
+lxml==2.2.3
+M2Crypto==0.20.2
+Magic-file-extensions==0.1
+Mako==0.3.4
+MarkupSafe==0.9.2
+matplotlib==0.99.1.1
+mglob==0.4
+netaddr==0.7.5
+nose==0.10.4
+numpy==1.10.4
+ofed-le-utils==1.0.3
+ordereddict==1.2
+paramiko==1.7.5
+pexpect==2.3
+policycoreutils-default-encoding==0.1
+pycrypto==2.0.1
+pycurl==7.19.0
+Pygments==1.1.1
+pygpgme==0.1
+pyOpenSSL==0.13.1
+python-dateutil==1.4.1
+python-default-encoding==0.1
+python-dmidecode==3.10.15
+python-ldap==2.3.10
+python-meh==0.11
+python-nss==0.16.0
+pythonbenchmark==1.4
+pytz===2010h
+pyxdg==0.18
+pyzmq==14.3.1
+requests==2.6.0
+rhsm==1.18.6
+root-numpy==4.7.2
+rootplot==2.2.1
+scdate==1.9.60
+sckdump==2.0.5
+scservices==0.99.45
+scservices.dbus==0.99.45
+setools==1.0
+simplegeneric==0.8
+six==1.9.0
+slip==0.2.20
+slip.dbus==0.2.20
+slip.gtk==0.2.20
+smbc==1.0
+SSSDConfig==1.13.3
+suds==0.4.1
+tornado==2.2.1
+urlgrabber==3.9.1
+urllib3==1.10.2
+virt-who==0.18
+virtinst==0.600.0
+yum-metadata-parser==1.1.2
```

Comment:
Yeah for sure, that's why I was asking how I could go about seeing what was *actually* required on a blank machine.

But I think the critical packages beyond "slc6" default are given here:

http://cmsonline.cern.ch/cms-elog/992603
http://cmsonline.cern.ch/cms-elog/992570

Packages needed I believe are:

NumPy versions 1.6, 1.7, 1.8, 1.9, 1.10 (default on slc6 in python 2.6.6 is 1.4 and this is too old to work with root_numpy)
root-numpy-4.7.2

Will update to include just these.